awx/ui_next/src/screens/Host/shared/HostForm.jsx

```diff
@@ -20,6 +20,8 @@ function HostForm({ handleSubmit, handleCancel, host, i18n }) {
     host ? host.summary_fields.inventory : ''
   );
 
+  const hostAddMatch = useRouteMatch('/hosts/add');
```

Comment:
We only want to show the inventory lookup if they are located at `/hosts/add`. The HostEdit, Inventory HostAdd, and Inventory HostEdit views should hide the inventory lookup.